Frontend/src/pages/admin/AdminStudents.vue

```diff
@@ -83,10 +110,81 @@ export default {
       .then(resp => {
         this.students = resp.data;
       });
+    this.$axios
+      .get("/coops/status", {
+        headers: {
+          Authorization: this.$store.state.token
+        }
```

Comment:
Don't need this yet technically since we haven't implemented actual authentication

Reply:
so i should remove them all? aren't we going to need this eventually

Reply:
You can leave it but just FYI for future stuff that you don't need to add this